tests/test_formats.py

```diff
@@ -892,6 +915,14 @@ def test_defaults_with_path(self):
             with tsinfer.load(filename) as other:
                 self.assertEqual(other, ancestor_data)
 
+    def test_bad_max_file_size(self):
+        with tempfile.TemporaryDirectory(prefix="tsinf_format_test") as tempdir:
+            filename = os.path.join(tempdir, "ancestors.tmp")
+            for bad_size in ['a', '', -1, [1, 3], np.array([1, 2])]:
+                self.assertRaises(
+                    (ValueError, TypeError), formats.AncestorData,
```

Comment:
Can we separate these out into two assertions please? This way I don't know which one throws a ValueError and which one throws a TypeError.

Reply:
Done